mappings/net/minecraft/block/SegmentedBlock.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_10596 net/minecraft/block/SegmentedBlock
```

Comment:
Shouldn't this be `Segmented` to be consistent with other block interfaces?

Reply:
True, also renamed `LandingBlock` to the more accurate `Falling` to reflect this